src/main/java/com/fabriccommunity/spookytime/doomtree/DoomTree.java

```diff
@@ -0,0 +1,102 @@
+package com.fabriccommunity.spookytime.doomtree;
+
+import java.util.function.Consumer;
+
+import com.fabriccommunity.spookytime.SpookyTime;
+import com.fabriccommunity.spookytime.doomtree.treeheart.DoomHeartBlockEntity;
+import com.fabriccommunity.spookytime.doomtree.treeheart.DoomTreeTracker;
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+
+import net.fabricmc.fabric.api.block.FabricBlockSettings;
+import net.fabricmc.fabric.api.event.server.ServerStopCallback;
+import net.fabricmc.fabric.api.tag.TagRegistry;
+import net.fabricmc.fabric.api.tools.FabricToolTags;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Material;
+import net.minecraft.block.entity.BlockEntityType;
+import net.minecraft.item.Item;
+import net.minecraft.sound.BlockSoundGroup;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.registry.Registry;
+
+public class DoomTree {
```

Comment:
why are you doing all the registry stuff in here instead of existing registry classes like SpookyBlocks and SpookyItems?

Reply:
Because every single time you do a merge I have to fix up my commits.  It can be reintegrated when I'm not changing things as regularly, but IMO organizing by feature set would have been preferable to organizing by registry type when you have many people making simultaneous changes.